extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NetBeansRunSinglePlugin.java

```diff
@@ -76,13 +77,16 @@ private void configureJavaExec(Project project, JavaExec je) {
             je.setArgs(asList(project.property(RUN_SINGLE_ARGS).toString().split(" ")));
         }
         if (project.hasProperty(RUN_SINGLE_JVM_ARGS)) {
+            // if `project' was as an implicit reference in CommandLineArgumentProvider, it would break configuration cache
+            // requirements - leads to a warning + failed build after run.
+            List<String> jvmArgs = asList(project.property(RUN_SINGLE_JVM_ARGS).toString().split(" "));
```

Comment:
Should the anonymous class be converted into a private static inner class, with a note that it must stay static?